manual/guides/travel.md

```diff
@@ -76,16 +75,34 @@ hotel where the conference is at. Airbnb can be unreliable.
 Please note that even while on travel, students, staff, and faculty are 
 representatives of the university and our ARFC research group.
 
-## 9. Fill out the TEM form
+### 9. After the trip, log in to the [Travel Expense Management (TEM) Services](https://www.obfs.uillinois.edu/tem-resources/) by clicking on the LOG IN button tomorrow. 
```

Comment:
9 and 10 may not be necessary since you have to log in later to submit the form prepared by tessa/kate.